packages/fxa-content-server/tests/functional/settings_v2/display_name.js

```diff
@@ -53,7 +54,7 @@ describe('display name', () => {
       selectors.SETTINGS_V2.AVATAR_DROP_DOWN_MENU.MENU_BUTTON,
       remote
     );
-    await testElementTextEquals(
+    await testElementTextInclude(
```

Comment:
After introducing https://github.com/projectfluent/fluent.js/wiki/Localized#markup component for the email in the drop down menu, it renders out an additional character before and after the email? I haven't been able to pinpoint what might be causing this but I suppose in meantime we can use `testElementTextInclude`.